subprojects/testing-jvm/src/integTest/groovy/org/gradle/testing/junitplatform/JUnitPlatformRetryUntilFailureIntegrationTest.groovy

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.testing.junitplatform;
+
+import org.gradle.testing.fixture.AbstractJvmRetryUntilFailureIntegrationSpec
+
+import static org.gradle.testing.fixture.JUnitCoverage.LATEST_JUPITER_VERSION
+
+class JUnitPlatformRetryUntilFailureIntegrationTest extends AbstractJvmRetryUntilFailureIntegrationSpec {
```

Comment:
I would like to see a non-Jupiter test for the platform as well.

Reply:
So you mean testing Junit4 with Junit platform? 

There are already Junit4 tests there, but yeah, not with the platform.